doc/releases/release-notes-4.1.rst

```diff
@@ -26,8 +26,8 @@
 
 .. _zephyr_4.1:
 
-Zephyr 4.1.0 (Working Draft)
-############################
+Zephyr 4.1.0
+############
 
 We are pleased to announce the release of Zephyr version 4.1.0.
```

Comment:
as discussed in a private chat - should we add a date here?

```suggestion
We are pleased to announce Zephyr version 4.1.0, released on March 7, 2025.
```

Reply:
I would not bother here to be totally honest, that information is in the git tag

Reply:
mainly I'd see this as a bit of information that will very easily get forgotten and merged incorrectly